src/ontology/mondo-edit.obo

```diff
@@ -564918,6 +564918,16 @@ is_a: MONDO:0005583 {source="OMIA:000844"} ! non-human animal disease
 intersection_of: MONDO:0005583 {source="OMIA:000844"} ! non-human animal disease
 intersection_of: MONDO:0700097 MONDO:0009950 {source="OMIA:000844"} ! cross-species analog pyruvate kinase deficiency of red cells
 
+[Term]
+id: MONDO:1010100
+name: myxedema coma
+def: "A rare complication of hypothyroidism characterized by altered mental status, hypothermia, and symptoms related to the slowing of other organ systems." [https://www.ncbi.nlm.nih.gov/books/NBK545193/, PMID:33083180, PMID:34194879, PMID:38279788]
+comment: Hypothyroid patients exhibit multiple physiologic alterations to compensate for the deficiency of thyroid hormone. Acute stressors like infection, myocardial infarction, cold exposure, and surgery in patients with poorly controlled hypothyroidism can overwhelm these homeostatic mechanisms and can lead to myxedema coma. It can also occur as a culmination of severe, longstanding hypothyroidism. PMID:34194879, https://www.ncbi.nlm.nih.gov/books/NBK545193/
+subset: rare
```

Comment:
this will be removed when the rare subset is updated. We need to ask Gioconda to add it to her list of rare diseases.

Reply:
rare disease subsets should not be added manually, unless these are manually curated from publications.